laterpay/application/Controller/Admin/TinyMCE.php

```diff
@@ -0,0 +1,196 @@
+<?php
+
+/**
+ * Laterpay TinyMCE controller.
```

Comment:
```suggestion
 * LaterPay TinyMCE controller.
```
Could we update the usage to this? we can keep all lowercase in variables and methods.